src/Core/Services/Implementations/LaunchDarklyFeatureService.cs

```diff
@@ -17,6 +18,16 @@ public class LaunchDarklyFeatureService : IFeatureService, IDisposable
         var ldConfig = Configuration.Builder(globalSettings.LaunchDarkly?.SdkKey);
         ldConfig.Logging(Components.Logging().Level(LogLevel.Error));
 
+        if (!string.IsNullOrEmpty(globalSettings.ProjectName))
+        {
+            ldConfig.ApplicationInfo(Components.ApplicationInfo()
+                .ApplicationId(globalSettings.ProjectName)
+                .ApplicationName(globalSettings.ProjectName)
+                .ApplicationVersion(AssemblyHelpers.GetGitHash() ?? AssemblyHelpers.GetVersion())
```

Comment:
🤔 Given that it will either have an hash we won't know anything about until we search for it in our repo or a version that will be easier to identify, do you know how this info will be used?

Reply:
I could have documented this a bit I suppose:

https://docs.launchdarkly.com/home/engineering-insights/config-deployment#set-the-application-version

The "version" here is not the visible portion and must be a commit SHA for the insights functions to work, and I wanted to be extra safe should this not be in a Git repo in some environments / builds. The line below is what will be visible in user interfaces.

Reply:
**slight improvement** Those docs suggest either the git hash or version tag. The pattern we follow is not to tag with the assembly version, but `v` + the version (`v2023.10.0`), for example.

Not likely to matter because I think the git version will be available.